src/styles/components/location.scss

```diff
@@ -1,6 +1,10 @@
 .location {
     background-color: var(--color-primary);
 
+    @include media('<=tablet') {
+        height: calc(100vh - 20.625rem);
```

Comment:
Solved for table. The issue persist on mobile. 

<img width="541" alt="Screen Shot 2021-03-09 at 9 57 57 PM" src="https://user-images.githubusercontent.com/11657599/110569436-84b11100-8122-11eb-80a7-3a0b85385484.png">

Reply:
TIP: add the header height to the section padding instead to change the height ;)